staticjinja/dep_graph.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding:utf-8 -*-
+
+"""
+Dependency graph for staticjina
+"""
+
+from copy import deepcopy
+
+
+class DepGraph(object):
+    """
+    A directed graph which will handle dependencies between templates and data
+    files in a site.
+
+    :param site:
+        A :class:`Site <Site>` object.
+
+    """
+    def __init__(self, site):
+        self.parents = dict(
+                (s, set()) for s in site.jinja_names + site.data_names
```

Comment:
We should avoid one letter variable names. They're both hard to read and hard to search and replace.